source/h2_stream.c

```diff
@@ -813,6 +827,11 @@ struct aws_h2err aws_h2_stream_on_decoder_headers_end(
             return s_send_rst_and_close_stream(stream, aws_h2err_from_last_error());
         }
     }
+    if (stream->thread_data.check_incoming_content_length) {
+        if (ignore_content_length) {
+            stream->thread_data.check_incoming_content_length = false;
```

Comment:
when you make future changes, make sure we still check for duplicate content-length headers